test/test_ops.py

```diff
@@ -231,69 +231,63 @@ def test_variant_consistency_jit(self, device, dtype, op):
                 with disable_autodiff_subgraph_inlining():
                     def fn(*inputs, **kwargs):
-                        attr = getattr(inputs[0], name)
-                        output = attr(*inputs[1:], **kwargs)
+                        output = func(*inputs, **kwargs)
                         return op.output_func(output)
 
                     # bfloat16 grad doesn't work for some operators
                     dtypes_to_grad_check = floating_and_complex_types_and(torch.half) \
-                        if op.skip_bfloat16_grad else floating_and_complex_types_and(torch.half, torch.bfloat16) 
+                        if op.skip_bfloat16_grad else floating_and_complex_types_and(torch.half, torch.bfloat16)
 
                     # Check scripted forward, grad, and grad grad
                     script_fn = create_script_fn(self, name, func_type, op.output_func)
 
-                    check_against_reference(self, 
+                    check_against_reference(self,
                                             script_fn,
-                                            fn, 
-                                            (*sample.input,) + sample.args, 
-                                            sample.kwargs, 
+                                            fn,
+                                            (*sample.input,) + sample.args,
+                                            sample.kwargs,
                                             no_grad=(dtype not in dtypes_to_grad_check))
 
                     # Check traced forward, grad, and grad grad
                     traced_fn = create_traced_fn(self, variant)
-                    check_against_reference(self, 
+                    check_against_reference(self,
                                             traced_fn,
-                                            fn, 
-                                            (*sample.input,) + sample.args, 
-                                            sample.kwargs, 
+                                            fn,
+                                            (*sample.input,) + sample.args,
+                                            sample.kwargs,
                                             no_grad=(dtype not in dtypes_to_grad_check))
 
                     # Check alias annotation schema for correctness (make
                     #   sure inputs that aren't supposed to be modified aren't)
-                    # Note: only runs in float32 and int64 because schema isn't affected by dtype, 
+                    # Note: only runs in float32 and int64 because schema isn't affected by dtype,
                     #   so running it on all dtypes is would be excessive
-                    if dtype in [torch.float32, torch.int32]:
-                        check_alias_annotation(name, (*sample.input,) + sample.args, sample.kwargs)
+                    if dtype in [torch.float32, torch.int32] and func_type == 'method':
```

Comment:
Why the restriction to only the method variant here?

Reply:
Ah yes, left over from debugging so just removed.